src/main/resources/bamm/meta-model/2.0.0/aspect-meta-model-shapes.ttl

```diff
@@ -468,6 +468,57 @@ bamm:EventShape
       ]
    ] .
 
+bamm:UnitShape
+   a sh:NodeShape ;
+   sh:targetClass bamm:Unit ;
+   sh:property [
+      sh:order 3 ;
+      sh:path bamm:commonCode ;
+      sh:datatype xsd:string ;
+      sh:maxCount 1;
+      sh:name "commonCode" ;
+      sh:description "Normalized short code for a unit." ;
+   ] ;
+   sh:property [
+      sh:order 4 ;
+      sh:path bamm:conversionFactor ;
+      sh:datatype xsd:string ;
+      sh:maxCount 1 ;
+      sh:name "conversionFactor" ;
+      sh:description "If the unit is derived from a reference unit, the human readable multiplication factor, e.g., \"10⁻²⁸ m²\"" ;
+   ] ;
+   sh:property [
+      sh:order 5 ;
+      sh:path bamm:numericConversionFactor ;
+      sh:datatype xsd:double ;
+      sh:maxCount 1 ;
+      sh:name "numericConversionFactor" ;
+      sh:description "If the unit is derived from a reference unit, the numeric multiplication factor, e.g., \"1.0E-28\"" ;
+   ] ;
+   sh:property [
+      sh:order 6 ;
+      sh:path bamm:quantityKind ;
```

Comment:
Is the Quantity Kind for a Unit not a required attribute?

Reply:
No, it is not. There are numerous units without a quantity kind:
![image](https://user-images.githubusercontent.com/1199036/149291857-b63b61b5-d780-45a9-907f-03ef69761e6b.png)